R/R/run.R

```diff
@@ -90,6 +90,28 @@ run_cmd <- function(flow_file, ...) {
     batch <- ""
   }
 
+  if ("step_functions" %in% names(flags)) {
+    sfn_cmd <- paste("step-functions", flags$step_functions)
+    for (subcommand in c("authorize", "generate_new_token", 
+                         "only_json", "running", "succeeded", 
```

Comment:
`running`, `succeeded`, `failed`, `timed_out`, `aborted` are args for the subcommand `step-functions list-runs`. It is not immediately obvious how we are processing a command say - `step-functions list-runs running`

Reply:
The user can do `Rscript flow.R step-functions list-runs --running`. Or 
```
run(step_functions = "list_runs", running=TRUE)
```